Makefile

```diff
@@ -37,11 +37,13 @@ test: node_modules ## Runs tests
 i18n-extract: ## Extract strings for translation from the source code
 	npm run mmjstool -- i18n extract-webapp
 
-node_modules: package.json package-lock.json
+node_modules: package.json package-lock.json packages/mattermost-components/package.json
```

Comment:
I don't know if this will be needed since changes to a subpackage's package.json should be reflected in package-lock.json

Reply:
Yes, unneeded